tests/test__init__.py

```diff
@@ -1,8 +1,10 @@
 import pytest
 from chaosazure import __get_cloud_env_by_name, init_client,\
     init_resource_graph_client
+from chaoslib.exceptions import InterruptExecution    
```

Comment:
Import ordering seems off in this file. Though it's a testing module it's less critical :)